common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -434,6 +434,18 @@ def validate(self):
         validation = super(LibraryContentDescriptor, self).validate()
         if not isinstance(validation, StudioValidation):
             validation = StudioValidation.copy(validation)
+        library_tools = self.runtime.service(self, "library_tools")
+        if not (library_tools and library_tools.can_use_library_content(self)):
+            validation.set_summary(
+                StudioValidationMessage(
+                    StudioValidationMessage.ERROR,
+                    _(
+                        u"Only courses that are created using the split "
```

Comment:
So far as I know, the only action they can take is creating a new course and hoping that it will be created under a compatible modulestore, which is what the initial phrasing told them to do. It would still take the work of a sysadmin to enable the split modulestore, though. I don't think a normal author has any... um... recourse... here. :)